### DISCUSSION TITLE
не работает канвас

### DISCUSSION THREAD

**Discussion by ivanoff13** [richrobber2/canvas-zoom#72] [Q&A] [answered]
после обновления  перестало работать расширение((

**Comment by daswer123** [comment]
Проверь этот issue возможно найдешь решение проблемы 
https://github.com/richrobber2/canvas-zoom/issues/63

**Reply by ivanoff13** [reply]
в контрол нете не работает,работает только во вкладке img2img impaiting

**Reply by ivanoff13** [reply]
Commit hash: baf6946e06249c5af9851c60171692c44ef633e0 есть эта надпись в консоле

**Reply by daswer123** [reply]
А так бы сразу и сказал :)
В вкладке settings, нужно включить интеграцию с controlnet

**Reply by daswer123** [reply]
https://github.com/richrobber2/canvas-zoom/discussions/42#discussioncomment-6102485

**Reply by ivanoff13** [reply]
она включена и не работает

**Reply by daswer123** [reply]
В контрол нет больше 1 вкладки? ( юнита )

**Reply by ivanoff13** [reply]
одна

**Reply by ivanoff13** [reply]
было же пару дней все ок!ты не исправлял ничего?

**Reply by daswer123** [reply]
Нет, но должно быть больше 1 вкладки, добавь через настройки котрол нета

**Reply by ivanoff13** [reply]
https://drive.google.com/file/d/1xyhHk1b3TNehrDDYEWsQs9XYveznx8nY/view?usp=sharing
вот скрин экрана

**Reply by daswer123** [reply]
Вижу, сейчас на работе, вечером разберусь

**Reply by daswer123** [reply, marked as answer]
Исправил, теперь все должно работать.
Спасибо за баг репорт :)

**Reply by ivanoff13** [reply]
да работает!только когда две вкладки отрываешь))
